### DISCUSSION TITLE
use tft object/class in different files

### DISCUSSION THREAD

**Discussion by epikao** [Bodmer/TFT_eSPI#1415] [General]
Hello, how can I include or define TFT_eSPI tft = TFT_eSPI(); in way that I can use the library not only in the main.cpp file but also in other included cpp-files (with tft.xx)?

**Comment by Bodmer** [comment]
You can pass a pointer to the tft class instance. See [TFT_eFEx example](https://github.com/Bodmer/TFT_eFEX/blob/master/examples/Jpeg_ESP32/Jpeg_ESP32.ino#L33) or the OpenWeather [example here](https://github.com/Bodmer/OpenWeather/blob/master/examples/TFT_eSPI_OpenWeather_LittleFS/TFT_eSPI_OpenWeather_LittleFS.ino#L80).
